### PR TITLE
Update changelog to prepare for 1.1.0

### PR DESCRIPTION
**Pull request by Technius**
This currently includes the changes made since the last changelog update. Please add suggestions if I missed anything / got anything wrong.

We still need to check against some of the older commits since November 2019.

### CODE REVIEW

**Comment by Febbe** [comment]
Fix Zoom Jitter #2743 missing (merged yesterday)

**Comment by Technius** [comment]
> Fix Zoom Jitter #2743 missing (merged yesterday)

That's a regression and should not be included as a change from 1.0.20.

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,28 +87,54 @@
...
+      direction (#2723).
+    * Added `Edit > Arrange` menu items and the corresponding actions for
+      rearranging selected elements (#2794).
+    * Fixed the confusing behavior of the `Apply to current/all pages` buttons
+      used to change the page backgrounds (#2730).
+    * Fixed bugs in element cloning, which previously could have caused elements
+      to become invalid (#2733, #2720, #2464).
+    * Fixed a bug where the thumbnailer would not correctly render icons in file
+      managers that sandbox their thumbnailers (#2738).
+    * Improved look of the the Preferences window (#2592).
```
Comment (rolandlo) [inline CHANGELOG.md:130]:
```suggestion
    * Improved look of the Preferences window (#2592).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,13 +59,21 @@
...
+    * Fixed touch drawing not working with the pen tool (#2123).
+* LaTeX tool
     * Reworked LaTeX tool implementation (#1952).
     * Added a new tab in the Preferences window for LaTeX configuration.
     * Added a `global template file` setting for custom LaTeX template files
       to be used when rendering LaTeX formulas (#1188).
     * Added a button in the Preferences window for testing LaTeX setup.
-* Splines
+    * Fixed a bug where closing the dialog before the initial render would crash
+      the application (#2728, 2798).
```
Comment (rolandlo) [inline CHANGELOG.md:75]:
```suggestion
      the application (#2728, #2798).
```

**Comment by Febbe** [comment]
> That's a regression fix and should not be included as a change from 1.0.20.

I thought this issue is there since at least 2 years, so it should be already in 1.0.20.

**Comment by Technius** [comment]
Ok, it looks like we didn't include the changes stemming from #2435 so we can just bundle up #2743 and the other related issues into the same changelog item.

**Comment by Technius** [comment]
I went through the old commits. I think this is almost everything (that is a _lot_ of changes!).

File: CHANGELOG.md [hunk 3]
```diff
@@ -45,31 +101,71 @@
     * Added a Preferences settings to preserve line width while resizing a
       selection (#2011)
     * Added a Preferences setting to change the snap grid size (#1920).
-    * Fixed a bug in the object selection algorithm (#2478)
-* Pen
+    * Added Preference settings to configure the grid size (#1920).
```
Comment (personalizedrefrigerator) [inline CHANGELOG.md:104]: Lines 103 and 104 both reference issue #1920... This seems a bit confusing.

Thank you for working on this!

Comment (Technius) [reply CHANGELOG.md:104]: Thanks for the catch.

**Review by Rmano — COMMENTED** [review]
Hmm... I am not a native English speaker, but I can't understand "to now selected pages during scrolling". Maybe you mean "to keep pages selected during scrolling"? Or "to select the current page during scrolling"? Thanks!

**Comment by Technius** [comment]
That was a typo, thanks for the catch.

**Comment by Technius** [comment]
Added some of the currently pending PRs that will go into 1.1.0.

**Comment by Technius** [comment]
Rebased and synchronized the PR list again.

**Comment by rolandlo** [comment]
I went through the changes of the last couple of months and added what was missing.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,74 +2,233 @@
 
 ## 1.1.0 / Nightly (Unreleased)
 
+This is a new major version of Xournal++ with many new features, improvements,
+and bug fixes thanks to over one year's worth of contributions from the
```
Comment (rolandlo) [inline CHANGELOG.md:6]:
```suggestion
This is a new major version of Xournal++ with many new features, improvements
and bug fixes thanks to over one year's worth of contributions from the
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,74 +2,233 @@
...
+    * MacOS: Dropped support for macOS High Sierra; minimal version is now Catalina (#2989)
+* Document viewing
+    * Changed page selection system to now select the current page during
+      scrolling (#1613, #1184).
+    * Improved page load performance and memory consumption with new page
+      preload mechanism (#2603).
+    * Fixed a bug where scrolling would not work with zoom fit enabled until the
+    zoom level changes (#2541).
+    * Fixed some bugs that preventing documents from being displayed on ARM and
+      32-bit devices (#2576).
```
Comment (rolandlo) [inline CHANGELOG.md:45]:
```suggestion
    * Fixed some bugs that prevented documents from being displayed on ARM and
      32-bit devices (#2576).
```

**Review by rolandlo — COMMENTED** [review]
I have only found 2 typos.

**Comment by rolandlo** [comment]
A line about #3209 (About dialog, AUTHORS file) should be added.

**Comment by Technius** [comment]
I updated the changelog again for the latest PRs.

**Review by rolandlo — APPROVED** [review]
@Technius Can you squash the commits, rebase on current master and solve the merge conflict (the MacOS startup crash fix should be included)? Then I think this PR is ready to be merged.

**Comment by rolandlo** [comment]
> @Technius Can you squash the commits, rebase on current master and solve the merge conflict (the MacOS startup crash fix should be included)? Then I think this PR is ready to be merged.

I did it myself, just needed to create a new PR #3241 . Therefore I will close this PR.

**Pull request closed without merging**